addons/loyalty_delivery/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     'summary': "Add a free shipping option to your rewards",
     'category': 'Sales',
     'version': '1.0',
-    'depends': ['loyalty', 'delivery'],
+    'depends': ['loyalty', 'delivery', 'sale_loyalty'],
```

Comment:
Additionnally, if we wanted to do that, then there would be no need to depends on `loyalty` any more if you  depend on `sale_loyalty`